src/content/en/updates/2018/03/present-web-pages-to-secondary-attached-displays.md

```diff
@@ -0,0 +1,268 @@
+
+    // Receiver page
+    
+    navigator.presentation.receiver.connectionList
+    .then(list => {
+      list.connections.map(connection => addConnection(connection));
+      list.addEventListener('connectionavailable', function(event) {
+        addConnection(event.connection);
+      });
+    });
+    
+    function addConnection(connection) {
+    
+      connection.addEventListener('message', function(event) {
+        console.log('Message: ' + event.data);
+        connection.send('Hey controller! I just received a message.');
+      });
+    
+      connection.addEventListener('close', function(event) {
+        console.log('Connection closed!', event.reason);
+      });
+    }
+
+A connection receiving a message will fire a "message" event you can listen to.
+The message can be a simple string, a Blob, an ArrayBuffer, or an
+ArrayBufferView. Sending it is as simple as calling `connection.send(message)`
+from the controller page or the receiver page.
+
+    // Controller page
+    
+    function onSendMessageButtonClick() {
+      presentationConnection.send('Hello!');
+    }
+    
+    presentationConnection.addEventListener('message', function(event) {
+      console.log('I just received ' + event.data + ' from the receiver.');
+    });
+
+Play with the sample at
+[https://googlechrome.github.io/samples/presentation-api/] to get a sense of
+how it works. I’m sure you’ll enjoy this as much as I do.
+
+## Samples and Demos
+
+Check out the official Chrome sample at
+[https://googlechrome.github.io/samples/presentation-api/] we’ve used for this
+article. 
+
+I would recommend the interactive Photowall demo as well at
+[https://googlechromelabs.github.io/presentation-api-samples/photowall]. This web
+app allows multiple controllers to collaboratively present a photo slideshow on
+a presentation display. Code source is available at
+[https://github.com/GoogleChromeLabs/presentation-api-samples].
+
+<figure>
+  <img src="/web/updates/images/2018/03/present-web-pages-to-secondary-attached-displays/hero.jpg"
+    alt="Photowall demo screenshot"/>
+  <figcaption>
+    <a href="https://www.flickr.com/photos/24100730@N08/3775286387/">
+      Photo
+    </a> by José Luis Mieza /
+    <a href="https://creativecommons.org/licenses/by-nc-sa/2.0/">CC BY-NC-SA 2.0</a>
+  </figcaption>
+</figure>
+
+## One moar thing
+
+Chrome has a "Cast" browser menu user can invoke at any time while visiting a
+website. If you want to control which presentation will be picked by default
+when user hits this menu, you should assign
+`navigator.presentation.defaultRequest` to a custom `presentationRequest`
+object created earlier.
+
+    // Make this presentation the default one when using the "Cast" browser menu.
+    navigator.presentation.defaultRequest = presentationRequest;
+
+## Dev Tips
+
+Go to the internal `chrome://inspect` page, select “Other”, and click the
+“inspect” link next to the currently presented URL to inspect the receiver page
+and debug it.
```

Comment:
Flip this sentence around.

'To inspect the receiver page and debug it, go to...'

Reply:
Done.